libs/utils/android/workloads/uibench.py

```diff
@@ -84,6 +84,11 @@ def run(self, out_dir, test_name, duration_s, collect=''):
         # Keep track of mandatory parameters
         self.out_dir = out_dir
         self.collect = collect
+        # For systrace collection, the duration is expected to be part of `collect` parameter
+        # This is because systrace capturing currently doesnt support CTRL+C stopping
+        # so tracingStop() call just waits instead.
+        if collect == 'systrace':
+            self.collect = 'systrace_' + str(duration_s)
```

Comment:
Looks like `duration_s` was actually required just because we needed to support systrace when there was not on-demand stop control.

Once we have #332 merged, we should probably completely remove `duration_s` and add instead a RE to capture the end of the test, as we already do for many other tests.
Look here for an example:
https://github.com/ARM-software/lisa/blob/master/libs/utils/android/workloads/jankbench.py#L152

Using RE we can stop on-demand both ftrace and systrace and that should be the "optimal" final solution.